hadoop-ozone/Jenkinsfile

```
@@ -48,7 +48,7 @@ node("ubuntu") {
         }, 'checkstyleResult')
 
         stageRunner('Acceptance', "acceptance", {
-             archiveArtifacts 'hadoop-ozone/dist/target/ozone-0.4.0-SNAPSHOT/smoketest/result/**'
+             archiveArtifacts 'hadoop-ozone/dist/target/ozone-*/smoketest/result/**'
```

```suggestion
             archiveArtifacts 'hadoop-ozone/dist/target/ozone-*/compose/result/**'
```

`acceptance.sh` no longer stores results in `smoketest/result`.  You can confirm this by running the script (though first you may want to tweak `hadoop-ozone/dist/src/main/compose/test-all.sh` to only run a single specific test, eg. `ozone-topology`).

---

@adoroszlai Thank you, that was helpful. I tried your idea to test it.